org.eclipse.winery.edmm/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2020 Contributors to the Eclipse Foundation
+  ~
+  ~ See the NOTICE file(s) distributed with this work for additional
+  ~ information regarding copyright ownership.
+  ~
+  ~ This program and the accompanying materials are made available under the
+  ~ terms of the Eclipse Public License 2.0 which is available at
+  ~ http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+  ~ which is available at https://www.apache.org/licenses/LICENSE-2.0.
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.eclipse.winery</groupId>
+        <artifactId>winery</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>org.eclipse.winery.edmm</artifactId>
+
+    <properties>
+        <checkstyle.config.location>../checkstyle.xml</checkstyle.config.location>
+    </properties>
+
+    <repositories>
+        <!-- allows to get the dependencies from github -->
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+    </repositories>
+    
+    <dependencies>
+        <dependency>
+            <groupId>com.github.ust-edmm.transformation-framework</groupId>
+            <artifactId>edmm-core</artifactId>
+            <version>7cf02a73e7ad0e8e09ba2668839123b71e538577</version>
```

Comment:
I tried to put `master` but jitpack seems not rebuilding the edmm-core. The new master it's not downloaded; maven uses an old version causing an error because it doesn't found a class.

Reply:
Ok, then we cannot merge until this has been resolved. We have to wait for a release of the transformation framework then.

Reply:
Can't you access jitpack web site and delete the master build while keeping the last release?

Reply:
We are anyhow not allowed to use the master build. We have to wait for the next release of the transformation framework.